### PR TITLE
fix reference issues in net46 by removing framework assembly refs

### PR DESCRIPTION
**Pull request by anurse**
part of Coherence-Signed/#395

### CODE REVIEW

File: test/Microsoft.AspNetCore.Hosting.FunctionalTests/Microsoft.AspNetCore.Hosting.FunctionalTests.xproj [hunk 1]
```diff
@@ -13,5 +13,8 @@
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```
Comment (BrennanConroy) [inline test/Microsoft.AspNetCore.Hosting.FunctionalTests/Microsoft.AspNetCore.Hosting.FunctionalTests.xproj]: ?

Comment (Tratcher) [inline test/Microsoft.AspNetCore.Hosting.FunctionalTests/Microsoft.AspNetCore.Hosting.FunctionalTests.xproj]: VS generated

File: src/Microsoft.AspNetCore.Server.IntegrationTesting/project.json [hunk 2]
```diff
@@ -46,12 +46,8 @@
...
       },
       "frameworkAssemblies": {
-        "System.Net.Http": "",
-        "System.Runtime": {
-          "type": "build"
-        },
-        "System.Xml": "",
-        "System.Xml.Linq": ""
+        "System.Xml": "4.0.0.0",
+        "System.Xml.Linq": "4.0.0.0"
```
Comment (BrennanConroy) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting/project.json]: Should we be concerned about the NETStandard.Library System.Xml.XDocument framework assembly reference to System.Xml.Linq?

File: test/Microsoft.AspNetCore.Hosting.FunctionalTests/Microsoft.AspNetCore.Hosting.FunctionalTests.xproj (hunk 1 above)
Comment (anurse) [inline test/Microsoft.AspNetCore.Hosting.FunctionalTests/Microsoft.AspNetCore.Hosting.FunctionalTests.xproj]: Random stuff VS adds sometimes when running tests. Better to leave it so it doesn't show as noise on a different commit.

**Pull request merged**